models/topics.models.js

```diff
@@ -84,3 +83,28 @@ exports.insertCommentOntoArticle = (article_id, author, body) => {
     }
   );
 };
+
+exports.updateArticle = (article_id, column, value) => {
+  return checkExists("articles", "article_id", article_id)
+  .then(
+    ({ exists }) => {
+      if (!exists) {
+        return Promise.reject({ status: 404, message: "404 - Not Found" });
+      }
+      const queryString = format(
+        `
+      UPDATE articles
+      SET %I = %I + $1 
+      WHERE article_id = $2
+      RETURNING *
+      ;`,
+        column, column
+      );
```

Comment:
You've tried to make this dynamic using 'updateArticle' and sending in a column, but then your queryStr only ever increases by the value passed in - which wouldn't work for any other column except one that has an integer. You wouldn't want this to update anything else other than votes, realistically, so I think it's safe to lose the 'column' parameter and then hardcode in votes :)

Reply:
I left it semi dynamic because I thought I might be getting ready to add more patch info in the next task... but then that didn't happen! I'll go over it in a bit